src/java.base/share/classes/java/lang/Character.java

```diff
@@ -63,7 +63,41 @@
  * <a href="http://www.unicode.org">http://www.unicode.org</a>.
  * <p>
  * Character information is based on the Unicode Standard, version 13.0.
- *
+ * <p>
+ * The Java platform has supported different versions of the Unicode
+ * Standard over time. Upgrades to newer versions of the Unicode Standard
+ * occurred in the following Java releases, each indicating the new version:
+ * <table class="striped">
+ * <caption style="display:none">Shows Java releases and supported Unicode versions</caption>
+ * <thead>
+ * <tr><th scope="col">Java release</th>
+ *     <th scope="col">Unicode version</th></tr>
+ * </thead>
+ * <tbody>
+ * <tr><td>Java SE 15</td>
+ *     <td>Unicode 13.0</td></tr>
+ * <tr><td>Java SE 13</td>
+ *     <td>Unicode 12.1</td></tr>
+ * <tr><td>Java SE 12</td>
+ *     <td>Unicode 11.0</td></tr>
+ * <tr><td>Java SE 11</td>
+ *     <td>Unicode 10.0</td></tr>
+ * <tr><td>Java SE 9</td>
+ *     <td>Unicode 8.0</td></tr>
```

Comment:
Do we really need the history in the API docs? Will will update this table if there is a MR of the JSR for Java 8 that moves to a new Unicode release?

Reply:
This was requested by Alex, and I thought it sounded reasonable and informative. I think if an MR upgrades the supported Unicode version, it should be listed here.

Reply:
Should some acknowledgement be made to the changes in Unicode support done by the Java SE 8 and Java SE 11 MRs with respect to extensions of the base Unicode version?

Reply:
I thought about that but decided not to include them here because the changes made in those MRs were not Unicode version upgrades (i.e., version stayed the same). Let me know if you think otherwise.